server-spi/src/main/java/org/keycloak/models/CIBAPolicy.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.models;
+
+import java.io.Serializable;
+
+import org.jboss.logging.Logger;
+
+public class CIBAPolicy implements Serializable {
+
+    protected static final Logger logger = Logger.getLogger(CIBAPolicy.class);
+
+    protected String backchannelTokenDeliveryMode = "poll";
```

Comment:
Do you expect sub-types or types within the same package to extend this class? Could them be `private` instead?

Reply:
@pedroigor No, I do not expect it.  As you suggested, `private` is appropriate. I will change these fields' visibility as `private`.